docs/building/internals.md

```diff
@@ -59,9 +59,7 @@ you likely need to delete the `gen` folders and re-run the tool.
 
 ### Type Catalog
 
-As a work-around for the lack of `GetAssemblies()`,
-our custom assembly load context takes a pre-generated catalog of C# types
-(for PowerShell type resolution).
+A pre-generated catalog of C# types is used in PowerShell to help type resolution.
```

Comment:
Line 71 and below needs to be updated. CorePsTypeCatalog.ps is not under AssemblyLoadContext anymore.

Reply:
Good catch. Will fix.

Reply:
Oh, forgot it. That one will be addressed by #5072. That PR is pending merging.

Reply:
I will merge this once #5072 is merged.

Reply:
I just merged  #5072 :)